src/dataframe.ts

```diff
@@ -1,11 +1,19 @@
 import { wrapPromise } from './promise.js'
-import { AsyncRow, Cells, Row, asyncRows, awaitRows } from './row.js'
+import { AsyncRow, Cells, asyncRows } from './row.js'
 
-export interface RowsArgs {
-  start: number
-  end: number
-  orderBy?: string
-}
+/**
+ * Function that gets values in a column.
+ *
+ * If start and end are provided, only get values in that range.
+ * Negative start and end are allowed.
+ *
+ * @param column Column name
+ * @param start Start index (inclusive)
+ * @param end End index (exclusive)
+ *
+ * @returns Values in the column
+ */
+export type GetColumn = ({ column, start, end }: {column: string, start?: number, end?: number}) => Promise<any[]>
```

Comment:
Eventually we may want to allow `any[]` OR TypedArrays as this will allow faster parsing from parquet. But I'd leave this as `any[]` for now, just making a note for the future.

Reply:
yes, exactly. As I mentioned here: https://github.com/hyparam/hightable/pull/51#issuecomment-2677883043, I chose this for now because hyparquet does not provide a way to get the TypedArrays (if I read correctly)

Reply:
related to https://github.com/hyparam/hyparquet/issues/65